src/cli/login.rs

```diff
@@ -21,6 +22,16 @@ use reqwest::StatusCode;
 
 use super::Console;
 
+#[cfg(test)]
+fn local_timezone() -> &'static chrono::Utc {
+    &chrono::Utc
+}
+
+#[cfg(not(test))]
+fn local_timezone() -> &'static chrono::Local {
+    &chrono::Local
+}
```

Comment:
I want to take another look at this before merging - I feel like we could do _something_ to improve it.

Reply:
Aha! There is syntax for what we want:
```rust
fn local_timezone() -> &'static impl TimeZone<Offset = impl Offset + Display> {
    &chrono::Utc
}
```